app/src/main/java/com/github/sdp_begreen/begreen/User.kt

```diff
@@ -0,0 +1,88 @@
+package com.github.sdp_begreen.begreen
+
+import android.os.Build
+import android.os.Parcel
+import android.os.Parcelable
+import androidx.annotation.RequiresApi
+
+//Need to be Parcelable to be passed as an argument to a fragment
+data class User (val id: Int, val name: String, val score: Int) : Parcelable, Comparable<User> {
+    var rating: Int = 0
```

Comment:
Declaring variables like that is easier but not secure. I recommend you to set them as private and create getters and setters. Quite annoying right ? This is why Swift is better than Kotlin ;)

Reply:
I don't know how insecure it is? could we speak about that with the others tommow morning?